# request
string info1
string info2
---
# response
bool success
